src/rf/js/src/home/components/library.js

```diff
@@ -493,12 +493,16 @@ var ImageMetadata = React.createBackboneClass({
 
     render: function() {
         var layer = this.getActiveLayer(),
-            image = layer && layer.getActiveImage();
+            image = layer && layer.getActiveImage(),
+            metadata = null;
 
         if (!image) {
             return null;
         }
 
+        metadata = JSON.parse(image.meta_json);
+        metadata = _.sortBy(metadata, 'label');
```

Comment:
I think this is not necessary. In all empty cases I tested, _.sortBy returns an empty array.

```javascript
_.sortBy([], 'label') = [ ]
_.sortBy(null, 'label') = [ ]
_.sortBy('', 'label') = [ ]
```